src/utilities/codemirror/seq-n-highlighter.ts

```diff
@@ -0,0 +1,67 @@
+import { syntaxTree } from '@codemirror/language';
```

Comment:
Are these files utility files for code mirror in general or do they add additional sequencing functionality? Ideally the `/codemirror` directory contains functionality that are generic enough to be reused for other aspects of the UI like constraints and scheduling editors once we move to all codemirror implementations.

If it's to add functionality, can we move all relevant files to either `/sequence-editor` or a sub-folder with a relevant name inside `/codemirror`

Reply:
This file is language specific, similar to the VML files I think we'll consider moving it out of core soon, but I'd prefer to do that in another refactor